src/main/resources/application.yml

```diff
@@ -40,8 +40,12 @@ app:
     oauth-client:
       resource: "${app.indexing-service.base-uri}"
     publisher-catalog-name: publisher_data
-  query-cleanup-interval: "*/10 * * ? * *" # Every 10 seconds
-  query-cleanup-timeout-in-seconds: 120
+  query-cleanup:
+    cron-interval: "*/10 * * ? * *" # Every 10 seconds
+    timeout-in-seconds: 120
+  query-job-cleanup:
+    cron-interval: "0 0 1 * * ?" # Every day at 1am
+    deletion-timeout-in-days: 1
```

Comment:
Why are we deleting entries from the table?
I would recommend keeping the entries at-least for 7 days, just in case we want to debug something in future. Wdyt?

Reply:
That was [Patricks suggestion](https://www.pivotaltracker.com/story/show/180907761/comments/229007984) as there's no need to store all entries in `query_job`. 

I agree with increasing the deletion timeout to 7 days